display_list/BUILD.gn

```diff
@@ -55,51 +55,74 @@ fixtures_location("display_list_benchmarks_fixtures") {
   assets_dir = "$target_gen_dir/"
 }
 
-if (enable_unittests) {
-  executable("display_list_benchmarks") {
-    testonly = true
+source_set("display_list_benchmarks_source") {
+  testonly = true
+
+  sources = [
+    "display_list_benchmarks.cc",
+    "display_list_benchmarks.h",
+  ]
 
-    sources = [
-      "display_list_benchmarks.cc",
-      "display_list_benchmarks.h",
+  deps = [
+    ":display_list",
+    ":display_list_benchmarks_fixtures",
+    "//flutter/benchmarking",
+    "//flutter/common/graphics",
+    "//flutter/fml",
+    "//flutter/testing:skia",
+    "//flutter/testing:testing_lib",
+    "//third_party/dart/runtime:libdart_jit",  # for tracing
+    "//third_party/skia",
+  ]
+
+  defines = []
+
+  if (is_android) {
+    libs = [
+      "android",
+      "EGL",
+      "GLESv2",
+    ]
+  } else {
+    # We only do software benchmarks on non-mobile platforms
```

Comment:
Shouldn't that be `else if !ios`?

Or just not use an else and have a separate `if !android && !ios`?

Reply:
good catch, thanks. I totally missed that.